components/storage/ComparisonCellContent.vue

```diff
@@ -20,6 +20,52 @@
         ]"
         ><i class="mdi mdi-speedometer"
       /></span>
+      <span
+        v-else-if="
+          typeof feature.class === 'string' && feature.class.startsWith('slow')
+        "
+        :class="[
+          'icon',
+          `is-size-${iconSize}`,
+          'has-text-danger',
+        ]"
+        ><i class="mdi mdi-speedometer-slow"
+      /></span>
+
+      <span
+        v-else-if="
+          typeof feature.class === 'string' && feature.class === 'hot'
```

Comment:
don't need to check if these are strings

Reply:
Perfect! Fixed! Thank you, @mcmcgrath13!!